pages/index.vue

```diff
@@ -144,6 +144,9 @@ export default {
   },
   async mounted() {
     // let emails = await this.$axios.$get("https://api.github.com/user/emails")
+    
+    //Set session id per user
+    document.cookie = `session_id=${Date.now()}`;
```

Comment:
Seems like we would want to set an expiry time for this cookie?

Reply:
Sure David. I initially had an approach in mind wherein we log the views per session (the cookie gets discarded once the user closes the browser). But since we're garbage collecting once every day, an expiry time of 1 day should be good.